tests/test_log.py

```diff
@@ -17,16 +17,37 @@
 
 import os
 import pytest
+from pathlib import Path
 
 from pyfmi.common.log import extract_xml_log, parse_xml_log
 from pyfmi.common.diagnostics import DIAGNOSTICS_PREFIX
 from pyfmi.test_util import Dummy_FMUModelME2
 from pyfmi.util import decode
+from pyfmi.fmi1 import FMUModelME1
+from pyfmi.fmi2 import FMUModelME2
+from pyfmi.fmi3 import FMUModelME3
 
 import numpy as np
 file_path = os.path.dirname(os.path.abspath(__file__))
 logs = os.path.join(file_path, "files", "Logs")
 
+this_dir = Path(__file__).parent.absolute()
+FMI1_REF_FMU_PATH = Path(this_dir) / 'files' / 'reference_fmus' / '1.0'
```

Comment:
Minor, but calling `.absolute()` or  creating a new `Path` instance of `this_dir` is not needed.